src/utils/vc/document-loader.js

```diff
@@ -1,19 +1,20 @@
 import axios from 'axios';
 import testContext from './contexts';
+import DIDResolver from '../../did-resolver'; // eslint-disable-line
 
 /**
  * Takes a resolver and returns a function that returns a document or throws an error when the document
  * cannot be found.
- * @param {Resolver} resolver - The resolver is optional but should be passed when DIDs need to be resolved.
- * @returns {function(*=): {documentUrl: *, document: null}}
+ * @param {DIDResolver} [resolver] - The resolver is optional but should be passed when DIDs need to be resolved.
+ * @returns {loadDocument} - the returned function
```

Comment:
Shouldn't `loadDocument` be function?

Reply:
it is a function, this defines that it returns the loadDocument function (that is referenced below at its definition)